README.md

```diff
@@ -1,7 +1,12 @@
 # The Smallest Starting Point
```

Comment:
Change the readme to something that means something for your app, not "instructions for the developers" but instructions for someone that is visiting your repo.